packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -131,7 +131,7 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
   private _wallet: Wallet | null = null;
 
   // Current connected account
-  private _account: AccountInfo | null = null;
+  private _account: AccountInfo | StandardAccountInfoInput | null = null;
```

Comment:
Same here, let's just use `AccountInfo` from the standard. We can convert it to the right type before setting its value